program/program_result.py

```diff
@@ -198,33 +188,46 @@ def fields_view_get(
         """Change the readonly fields depending on the groups
         and the settings of the levels
 
-        Look for the strings "[('state', '!=', 'draft')]"
-        exactly in the xml view and replace it with
-        "[('state', 'not in', ('draft', 'validated'))]"
+        Non-employees cannot edit the form except for changing the state
 
         Hide intervention_id in all but last level results
         Hide pages and fields according to the config parameters of
         the level
         """
         def hide_elem(elem, invisible="true"):
+            """Set field element's invisibility status
+
+            :param lxml.etree.Element elem: View's field
```

Comment:
That is not compatible with sphinx doc.

If you want to make it compatible please change to
```
:param elem: View's field
:type elem: lxml.etree.Element
```

or more simple:
```
:param elem: lxml.etree.Element, View's field
```